bibliopixel/remote/control.py

```diff
@@ -104,13 +110,32 @@ def __init__(self, layout, animations,
         self.handlers = {
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
-            'get_config': self.get_config
+            'get_config': self.get_config,
+            'trigger_animation': self.run_animation
         }
 
+        self.trigger_procs = {}
+        self.triggers = {}
+        for t in triggers:
+            typename = t['typename']
```

Comment:
What happens if `typename` isn't in the triggers?  This is an error that can be caused by bad user data, so it shouldn't cause the program to crash!

You should report this and then, probably, just skip that trigger rather than failing.